syntax = "proto3";

package helloworld;
option go_package = "github.com/DotAScripter/helm-demo-project/greetings/proto/helloworld";
option java_package = "com.hdp.jpod.proto";

// The greeting service definition.
service Greeter {
  // Sends a greeting
  rpc SayHello (HelloRequest) returns (HelloReply) {}
}

// The request message containing the user's name.
message HelloRequest {
  string name = 1;
}

// The response message containing the greetings
message HelloReply {
  string message = 1;
}
